### PR TITLE
OKTA-440989 - Add sauceLabs script 

### PR DESCRIPTION
**Pull request by yuliialysenko-okta**

### CODE REVIEW

File: .bacon.yml [hunk 1]
```diff
@@ -61,4 +61,11 @@ test_suites:
       timeout: '60'
       script_name: publish
       criteria: MERGE
+      queue_name: small
+    - name: e2e-saucelabs 
+      script_path: ../okta-auth-js/scripts
+      sort_order: '8'
+      timeout: '60'
```
Comment (shuowu) [inline .bacon.yml:68]: nit: let's start with a shorter timeout, like `20`, so we are not waiting on bacon for some edge cases.

File: test/e2e/sauce.wdio.conf.js [hunk 2]
```diff
@@ -98,16 +114,34 @@ exports.config = {
...
+        // {browserName: 'chrome', platform: 'OS X 10.13', version: 'latest'}
+        {
+            browserName: 'MicrosoftEdge',
+            browserVersion: 'latest',
+            platformName: 'Windows 10',
+            'sauce:options': {
+                recordScreenshots: 'false'
+            }
+        }
+            //{
```
Comment (shuowu) [inline test/e2e/sauce.wdio.conf.js:135]: are we going to test IE 11?

File: test/e2e/sauce.wdio.conf.js [hunk 3]
```diff
@@ -161,31 +195,42 @@ exports.config = {
...
         sauceConnect: true,
-        sauceConnectOpts: { noAutodetect: false }
+        sauceConnectOpts: { noAutodetect: false,
+            tunnelIdentifier: 'e2e-tunnel'
+        }
       }],
     ],
     // Framework you want to run your specs with.
-    // The following are supported: Mocha, Jasmine, and Cucumber
+    // The following are supported: Mocha, Jasmine, and Cucumber§
```
Comment (shuowu) [inline test/e2e/sauce.wdio.conf.js:204]: nit: remove the special character

File: test/e2e/specs/login.js [hunk 4]
```diff
@@ -33,8 +33,8 @@ describe('E2E login', () => {
 
   flows.forEach(flow => {
     describe(flow + ' flow', () => {
-      async function bootstrap(options = {}) {
-        (flow === 'pkce') ? await openPKCE(options) : await openImplicit(options);
+      async function bootstrap(options = {}, openInNewWindow) {
```
Comment (shuowu) [inline test/e2e/specs/login.js:36]: add openInNewWIndow to options?

File: test/e2e/specs/sso.js [hunk 5]
```diff
@@ -70,7 +70,7 @@ describe('SSO', () => {
     // We should not be signed in on Okta page
     await openOktaHome();
     await OktaLogin.waitForLoad();
-
+    await browser.closeWindow();
```
Comment (shuowu) [inline test/e2e/specs/sso.js:73]: what's this line for?

File: test/e2e/sauce.wdio.conf.js (hunk 3 above)
Comment (yuliialysenko-okta) [reply test/e2e/sauce.wdio.conf.js]: done

File: test/e2e/specs/login.js (hunk 4 above)
Comment (yuliialysenko-okta) [reply test/e2e/specs/login.js]: done

**Review by shuowu — APPROVED** [review]
